packages/design-system/src/components/contextMenu/menuItem.js

```diff
@@ -206,7 +206,7 @@ export const MenuItemProps = {
     display: PropTypes.string,
     title: PropTypes.string,
   }),
-  Icon: PropTypes.func,
+  Icon: PropTypes.oneOfType([PropTypes.node, PropTypes.object]),
```

Comment:
This was throwing a proptype error 😕. Create react app uses [svgr](https://react-svgr.com/) under the hood to render svgs, but apparently they're still not a `PropTypes.node`. Feels like `PropTypes.object` is similar to a blanket type like `any` so it'd be good to tighten this up.